### PR TITLE
add usage info in README quick start

### PR DESCRIPTION
**Pull request by etpinard**
@mdtusz doc Tuesday continues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ chart types, including 3D charts, statistical graphs, and SVG maps.
 * [Documentation](#documentation)
 * [Contributing](#contributing)
 * [Community](#community)
-* [Clients for R, Python, and MATLAB](#clients-for-r-python-and-matlab)
+* [Clients for R, Python, and MATLAB](#clients-for-r-python-node-and-matlab)
```
Comment (mdtusz) [inline README.md]: Add node to the text too?

**Comment by mdtusz** [comment]
Up to you on the `node` thing, but💃  either way.

File: README.md (hunk 1 above)
Comment (etpinard) [inline README.md]: done already: https://github.com/plotly/plotly.js#clients-for-r-python-node-and-matlab

Comment (mdtusz) [inline README.md]: Meant in the link ⬆️

Comment (etpinard) [inline README.md]: ah good :eyes: my bad

Comment (etpinard) [inline README.md]: done.

**Pull request merged**